packages/esm-ohri-pmtct-app/src/config-schema.ts

```diff
@@ -16,6 +16,103 @@ export const configSchema = {
       laborAndDelivery: '6dc5308d-27c9-4d49-b16f-2c5e3c759757',
       infantPostnatal: 'af1f1b24-d2e8-4282-b308-0bf79b365584',
       motherPostnatal: '269bcc7f-04f8-4ddc-883d-7a3a0d569aad',
+      mnchEncounterType_UUID: '3eb24129-9f7d-4372-a14e-c2cf577e5db1',
+      clientsEnrolledToCar: '51bec6f7-df43-426e-a83e-c1ae5501372f',
+      vLResultsConcept: '1305AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      PTrackerIdentifierType: '4da0a3fe-e546-463f-81fa-084f098ff06c',
+      artUniqueNumberType: '9d6d1eec-2cd6-4637-a981-4a46b4b8b41f',
+      pTrackerIdConcept: '6c45421e-2566-47cb-bbb3-07586fffbfe2',
+    },
+  },
+  labour: {
+    _type: Type.Object,
+    _description: 'Labour & Delivery Concept and uuid.',
+    _default: {
+      clientsEnrolledForTb: '98fd11ba-cb4d-46f7-9b82-40d49949c7ef',
+      labourAndDeliveryEncounterType: '2678423c-0523-4d76-b0da-18177b439eed',
+      infantStatusAtBirthConcept: '159917AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      dateOfDeliveryConcept: '5599AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      bookedForAncConcept: '1719AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ancHivResultConcept: 'c5f74c86-62cd-4d22-9260-4238f1e45fe0',
+      hivStatusAtDeliveryConcept: 'c5f74c86-62cd-4d22-9260-4238f1e45fe0',
+      artInitiationConcept: '6e62bf7e-2107-4d09-b485-6e60cbbb2d08',
+      birthCountConcept: '1568AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      visitDate: '163260AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      hivTestStatus: '164401AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      artLinkage: '6e62bf7e-2107-4d09-b485-6e60cbbb2d08',
+      recenctViralLoad: '163310AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      motherStatusConcept: '1856AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      ancVisitConcept: '160446AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      infantDeliveryGroupingConcept: '1c70c490-cafa-4c95-9fdd-a30b62bb78b8',
+      infantPTrackerIdConcept: '164803AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+      infantDateOfBirth: '164802AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA',
+    },
+  },
+  infantpnc: {
+    _type: Type.Object,
+    _description: 'Infant Postnatal encounterTypes and concepts uuid.',
+    _default: {
+      infantPostnatalEncounterType: 'af1f1b24-d2e8-4282-b308-0bf79b365584',
```

Comment:
@lucyjemutai please combine these concepts to a single `obsConcepts` because some of them are used in more than one more form and components for example `clientsEnrolledForTb` is being used 3 times and you've defined it 3 times in this file. Please refer to the tb esm config for reference

Reply:
My bad. I have noted that, Thanks